server/server.js

```diff
@@ -22,7 +22,8 @@ if (process.env.NODE_ENV === 'production') {
   app.use(express.static(path.join(__dirname, '../client/build')));
 }
 
-app.get('/*', (req, res) => {
+// Having /* causes GraphQL to not function properly
```

Comment:
Adding the * is what solved the Apollo errors we were getting on the live site. What GraphQL error is happening because of this?